rebar.config

```diff
@@ -27,30 +27,13 @@
 
 {deps, [
   {sumo_db, "0.6.4"},
-  {riakc, "2.4.1"},
+  {riakc, {git, "https://github.com/basho/riak-erlang-client.git", {tag, "2.5.2"}}},
```

Comment:
2.5.2 pushed today. I also added @lukebakken and @alexmoore as owners, so hopefully they can keep it more up to date in the future!